sotodlib/coords/pointing_model.py

```diff
@@ -146,10 +149,11 @@ def model_sat_v1(params, az, el, roll):
 
     The implemented model parameters are:
 
-      - bs_{xi,eta}0: within the focal plane (i.e. relative to the
+      - fp_rot_{xi,eta}0: within the focal plane (i.e. relative to the
         corrected boresight), the center of rotation of the boresight.
-        Radians.
-      - enc_offset_{az,el,boresight}: encoder offsets, in degrees.
+         Radians.
+      - fp_offset_{xi,eta}0: Offset of focal plane's rotational center relative to position in focal plane that lies along the optical axis. Corrects for "collimation error"
+      - enc_offset_{az,el,boresight}: encoder offsets, in Radians.
       - base_tilt_{cos,sin}: base tilt coefficients, in radians.
 
```

Comment:
Add az_rot.

Reply:
Done: "az_rot: Dimensionless parameter describing a linear dependence of Az on El."